app/routes/next/explore/events.tsx

```diff
@@ -74,13 +73,22 @@ export const periodOfTimeValues = [
   "nextMonth",
   "past",
 ] as const;
+const stageValues = ["all", "hybrid", "online", "on-site"] as const;
```

Comment:
Should we use the slugs from the database to keep a single source of truth?

And only add the option "all" (which is not present in the db) to the db stages and as default value in the schema?

Reply:
I've changed this